### ISSUE TITLE
untimeError: Expected object of type torch.cuda.FloatTensor but found type torch.FloatTensor for argument #4 'other'

### ISSUE THREAD

**Issue by HAWLYQ** [ruotianluo/ImageCaptioning.pytorch#87]
Hi, when I evaluate a pretrained model named `att2in', I get a Runtime Error.   I think there is nothing wrong with my environment (py2.7  torch0.4.1).  Could you give some suggestion? 
![ImageCaptionError](https://user-images.githubusercontent.com/12978686/55541722-212d3600-56f8-11e9-99d8-9c7688f12e93.png)

**Comment by HAWLYQ** [comment]
I've solved this error by move '.cuda()' in 'img = torch.from_numpy(img.transpose([2, 0, 1])).cuda()' to 'img = Variable(preprocess(img))'. I think it's because the mean and std in normalization is not in GPU.
